lldb/source/Core/Module.cpp

```diff
@@ -1510,6 +1510,18 @@ bool Module::SetLoadAddress(Target &target, lldb::addr_t value,
   return false;
 }
 
+bool Module::SetLoadAddressByType(Target &target, lldb::addr_t value,
+                            bool value_is_offset, bool &changed, int type_id) {
+  ObjectFile *object_file = GetObjectFile();
+  if (object_file != nullptr) {
+    changed = object_file->SetLoadAddressByType(target, value, value_is_offset, type_id);
+    return true;
+  } else {
+    changed = false;
+  }
```

Comment:
Tip for later PRs, https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements

(but don't bother with style changes at this point of course)